src/Svg.Model/Drawables/Elements/TextDrawable.cs

```diff
@@ -110,7 +110,7 @@ internal string PrepareText(SvgTextBase svgTextBase, string value)
     {
         value = ApplyTransformation(svgTextBase, value);
         value = new StringBuilder(value).Replace("\r\n", " ").Replace('\r', ' ').Replace('\n', ' ').Replace('\t', ' ').ToString();
-        return svgTextBase.SpaceHandling == XmlSpaceHandling.Preserve ? value : s_multipleSpaces.Replace(value.Trim(), " ");
+        return svgTextBase.SpaceHandling == XmlSpaceHandling.Preserve ? value : s_multipleSpaces.Replace(value.TrimStart(), " ");
```

Comment:
Are you sure this is correct white space handling for all cases ?

Reply:
I compared with how Chrome renders

Reply:
using that SVG I provided